internal/core/thirdparty/tantivy/tantivy-binding/src/util.rs

```diff
@@ -28,3 +30,19 @@ pub fn free_binding<T>(ptr: *mut c_void) {
         drop(Box::from_raw(real));
     }
 }
+
+pub(crate) fn get_stop_words_list(value: &json::Value) -> Result<Vec<String>, TantivyError>{
```

Comment:
simply `get_string_list` or even generic `get_json_value_list`. returned error should also be generic: `json::Value is not a list of <type>`, and the caller to add info about stop word to the error msg.